app/assets/stylesheets/general/_form.scss

```diff
@@ -312,3 +316,75 @@ input[type="file"] {
   font-size: $font-size-small;
   color: $text-color-dark;
 }
+
+.form-tags-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 0 5px 5px 5px;
+}
+
+.form-tags-input {
```

Comment:
These class names don't necessarily follow BEM. The wrapper should be called something like `.form-tags` and this child would be `.form-tags__input`, and similarly for other classes used here.

I don't super mind, so feel free this ignore this one if you don't feel like changing it.